sky/clouds/service_catalog/data_fetchers/fetch_azure.py

```diff
@@ -244,11 +244,79 @@ def get_additional_columns(row):
         axis='columns',
     )
 
+    def create_gpu_map(df):
+        # Map of Azure's machine with GPU to their corresponding memory
+        # Result is hard-coded since Azure's API to not return such info
+        # may be outdated so need to be maintained
+        gpu_map = {
+            'Standard_NC6': 12,
+            'Standard_NC12': 24,
+            'Standard_NC24': 48,
+            'Standard_NC24r*': 48,
+            'Standard_NC6s_v2': 16,
+            'Standard_NC12s_v2': 32,
+            'Standard_NC24s_v2': 64,
+            'Standard_NC24rs_v2*': 64,
+            'Standard_NC6s_v3': 16,
+            'Standard_NC12s_v3': 32,
+            'Standard_NC24s_v3': 32,
+            'Standard_NC4as_T4_v3': 16,
+            'Standard_NC8as_T4_v3': 16,
+            'Standard_NC16as_T4_v3': 16,
+            'Standard_NC64as_T4_v3': 64,
+            'Standard_NC24ads_A100_v4': 80,
+            'Standard_NC48ads_A100_v4': 160,
+            'Standard_NC96ads_A100_v4': 320,
+            'Standard_ND96asr_v4': 40,
+            'Standard_ND96amsr_A100_v4': 80,
+            'Standard_ND6s': 24,
+            'Standard_ND12s': 48,
+            'Standard_ND24s': 96,
+            'Standard_ND24rs*': 96,
```

Comment:
why is there a `*`?

Reply:
Its one of the instance type offered by azure. 
https://learn.microsoft.com/en-us/azure/virtual-machines/nc-series